tidy3d/plugins/microwave/rf_material_library.py

```diff
@@ -0,0 +1,68 @@
+"""Holds dispersive models for several commonly used RF materials."""
+
+from typing import List
+
+import pydantic.v1 as pd
+
+from ...components.base import Tidy3dBaseModel
+from ...components.medium import PoleResidue
+from .rf_material_reference import ReferenceData, material_refs
+
+
+class AbstractMaterialItem(Tidy3dBaseModel):
+    """Reference, and data_source for a variant of a material."""
+
+    reference: List[ReferenceData] = pd.Field(
+        None,
+        title="Reference information",
+        description="A list of references related to this variant model.",
+    )
+
+
+class MaterialItem(AbstractMaterialItem):
+    """Reference, data_source, and material model for a variant of a material."""
+
+    medium: PoleResidue = pd.Field(
+        ...,
+        title="Material dispersion model",
+        description="A dispersive medium described by the pole-residue pair model.",
+    )
+
+
+rf_material_library = dict(
+    Rogers3010=MaterialItem(
+        medium=PoleResidue(
+            eps_inf=2.080628548409516,
+            poles=[
+                (
+                    (-14424652487575.385 + 0j),
+                    (40799026234878.12 - 0j),
+                ),
+            ],
+            frequency_range=(10e9, 10e9),
```

Comment:
this looks off: empty frequency_range?

Reply:
The parameters are fitted at a single frequency of 10 GHz. Shall we set it as `frequency = 10e9`?

Reply:
well, we shouldn't include material in material library that is only valid at a single frequency, as most simulation is broadband in FDTD. And you don't really need fitting for single frequency. If the reference doesn't provide a frequency range, try to come up with a frequency range.

Reply:
Yeah, I just did some search, for microwave circuits the frequency typically ranges from 1GHz to 30GHz, I believe this could be a good range to extract parameters.

Reply:
sounds good!

Reply:
The range of 1-30 GHz is typical for most Rogers laminates. For other materials targeted at lower frequencies, like FR-4 in the 1-2 GHz range, we can handle them accordingly.

Reply:
in that case, you can just use frequency ranges for those different materials. For the same material, we can also provide different variants targeting at different frequency range.